src/nvim/ex_docmd.c

```diff
@@ -155,6 +155,8 @@ struct dbg_stuff {
 # include "ex_cmds_defs.generated.h"
 #endif
 
+static const int command_count = 539;
```

Comment:
This can't be right because Nvim has `:checkhealth`. But I see that this is removed later, so doesn't matter.